client/my-sites/themes/theme-activation-confirmation-modal.jsx

```diff
@@ -204,22 +235,29 @@ export default connect(
 	( state ) => {
 		const siteId = getSelectedSiteId( state );
 		const installingThemeId = getPreActivateThemeId( state );
+		const activeThemeId = getActiveTheme( state, siteId );
 
 		return {
 			siteId,
+			activeThemeId,
+			activeThemeName: get( state, 'themes.queries.wpcom.data.items.' + activeThemeId + '.name' ),
```

Comment:
It looks like we have a selector for this: https://github.com/Automattic/wp-calypso/blob/trunk/client/state/themes/selectors/get-active-theme.js

And I think we already have the data for `installingThemeName` as part of `theme`, rather than using `get`.

Reply:
`getActiveTheme` returns the ID rather than the name (ie. `twentytwentyone` instead of `Twenty Twenty-One`), but I think that you're right about there being a much better way to do this using `getCanonicalTheme` - done so now! :)